# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/blindcoder/kterm/malloc.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./button.c.orig	2005-04-10 00:30:30.676719528 +0200
+++ ./button.c	2005-04-10 00:31:05.481428408 +0200
@@ -41,7 +41,7 @@
 #include "error.h"
 #include "menu.h"
 
-extern char *malloc();
+/* extern char *malloc(); */
 
 extern void DoSecureKeyboard();
 
--- ./charproc.c.orig	2005-04-10 00:31:27.290112984 +0200
+++ ./charproc.c	2005-04-10 00:31:49.660712136 +0200
@@ -102,8 +102,8 @@
 extern XtAppContext app_con;
 extern Widget toplevel;
 extern void exit();
-extern char *malloc();
-extern char *realloc();
+/* extern char *malloc();  */
+/* extern char *realloc(); */
 extern fd_set Select_mask;
 extern fd_set X_mask;
 extern fd_set pty_mask;
